INSTALL.rst

```diff
@@ -1,14 +1,21 @@
 Installation
 ============
 
-Manual
-------
+Step-by-step
+------------
+
+Install `ffmpeg` and ensure that `ffprobe` is in your PATH:
+
+.. code-block:: console
+
+    $ ffprobe
+    ... ffprobe version 3.3.3 ...
 
 Prepare the environment
 
 .. code-block:: console
 
-    $ npm install -g node-sass clean-css requirejs uglify-js
+    $ ./scripts/setup-npm.sh
     $ mkvirtualenv cds3
     (cds3)$ cdvirtualenv ; mkdir src ; cd src
     (cds3)$ git clone -b cdslabs https://github.com/CERNDocumentServer/cds.git
```

Comment:
Hey, could you, please, modify this line to:
`(cds3)$ git clone https://github.com/CERNDocumentServer/cds.git`
as the `cdslabs` branch is not updated since a long time, so the default (`cdslabs_qa`) should be used instead?